template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/AzureAdB2C/AzureAdB2CServiceCollectionExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static IServiceCollection AddAzureAdB2CAuthentication(this IServiceCollec
         private class BindAzureAdB2COptions : ConfigureOptions<AzureAdB2COptions>
         {
             public BindAzureAdB2COptions(IConfiguration config) : 
-                base(options => config.GetSection("Microsoft:AspNetCore:Authentication:AzureAdB2C").Bind(options))
+                base(options => config.GetSection("Authentication:AzureAdB2C").Bind(options))
```

Comment:
Dan do we want to move to the correct schema now or later?  Maybe we should be fixing the config to have the Microsoft/AspNetCore sections, I missed that they were missing...